jumanji/env_viewer.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2022 InstaDeep Ltd. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Abstract environment viewer class."""
+
+import abc
+from typing import Generic, Optional, Sequence
+
+import matplotlib
+from numpy.typing import NDArray
+
+from jumanji.env import State
+
+
+class Viewer(abc.ABC, Generic[State]):
```

Comment:
I wonder, if we assume matplotlib, should we call this `MatplotlibViewer` or something?

Reply:
Agreed to stick to `Viewer`